pkg/reconciler/clusterducktype/controller.go

```diff
@@ -51,9 +51,15 @@ func NewController(
 
 	// Watch custom resource definitions.
 	grDt := func(obj interface{}) {
+		r.resyncResourceMapper(ctx)
 		impl.GlobalResync(ducktypeInformer.Informer())
 	}
 	crdInformer.Informer().AddEventHandler(controller.HandleAll(grDt))
 
 	return impl
 }
+
+type ResourceMapping struct {
```

Comment:
is this used somewhere?

Reply:
oh shoot! no, it is og cruft. will delete.